tests/processes/ProcessingTest.py

```diff
@@ -118,15 +118,15 @@ def test_run(self):
         global_result = global_results[0]
 
         self.assertRegex(repr(local_result),
-                         "<Result object\\(id={}, origin='LocalTestBear', aff"
+                         '<Result object\\(id={}, origin='LocalTestBear', aff'
                          'ected_code=\\(\\), severity=NORMAL, confidence=100'
-                         ", message='test msg', aspect=NoneType\\) at "
+                         ', message='test msg', aspect=NoneType\\) at '
                          '0x[0-9a-fA-F]+>'.format(hex(local_result.id)))
         self.assertRegex(repr(global_result),
-                         "<Result object\\(id={}, origin='GlobalTestBear', "
+                         '<Result object\\(id={}, origin='GlobalTestBear', '
                          'affected_code=\\(.*start=.*file=.*section_executor_'
                          'test_files.*line=None.*end=.*\\), severity=NORMAL, c'
-                         "onfidence=100, message='test message', "
+                         'onfidence=100, message='test message', '
```

Comment:
All of you modified lines from L 121-129 need double quotes because the single quotes are part of the string